forge-gui/res/cardsfolder/upcoming/deep_cavern_bat.txt

```diff
@@ -0,0 +1,12 @@
+Name:Deep-Cavern Bat
+ManaCost:1 B
+Types:Creature Bat
+PT:1/1
+K:Flying
+K:Lifelink
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigReveal | TriggerDescription$ When CARDNAME enters the battlefield, look at target opponent's hand. You may exile a nonland card from it until CARDNAME leaves the battlefield.
+SVar:TrigReveal:DB$ RevealHand | Defined$ Targeted | ValidTgts$ Opponent | SubAbility$ ExileCard
```

Comment:
`Look$`

Reply:
Does this mean it needs to be done using choosecard with remembered ?

Reply:
I don't know right now if that would work vs. land only hands
you could check for similar scripts, maybe the AI understands one variant better